ext/civicrm_admin_ui/managed/SavedSearch_Contact_Summary_Memberships.mgd.php

```diff
@@ -108,19 +118,26 @@
             ],
             [
               'type' => 'field',
-              'key' => 'IF_contribution_recur_id_contribution_recur_id',
-              'dataType' => 'String',
-              'label' => E::ts('Auto-Renew'),
-              'sortable' => TRUE,
-              'title' => NULL,
-              'cssRules' => [],
-              'rewrite' => '',
+              'key' => 'Membership_ContributionRecur_contribution_recur_id_01.auto_renew',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Auto Renew'),
+              'sortable' => FALSE,
+              'rewrite' => '{ts}{/ts}',
```

Comment:
That trick works but so does this:
```suggestion
              'rewrite' => ' ',
```

Reply:
Yes. It doesn't allow this while configuring the searchkit on UI for some reason. Agree this works with mgd. Thanks 👍